bots/create-bot-users/README.md

```diff
@@ -1,8 +1,8 @@
-# Creating Bot Users
+# Create Bot Users
```

Comment:
> # Create Bot Users

# Create a Bot User

(you use 'a' below)

Reply:
This file should be deleted as redundant